mqtt/mqtt-broker/config/default.json

```diff
@@ -1,4 +1,17 @@
 {
+    "transports": [
+        { 
+            "tcp": {
+                 "address": "0.0.0.0:1883"
+            } 
+        },
+        { 
+            "tls": { 
+                "address": "0.0.0.0:8883", 
+                "identity_path": "identity.pfx" 
```

Comment:
Nit: Can we call this something like server_cert or so? Identity seems confusing here.

Reply:
then cert_path, server_cert_path, srv_cert_path? i think the path should be there

Reply:
renamed srv_cert_path

Reply:
I think we need more consistent here: you have address (full form), but srv_cert_path (several short words).
I suggest we call it `certificate` and in rust code it will be `certificate: PathBuff`. So no confusion about what is certificate type

Reply:
in the config it is not immediately clear that it should be a path, but "certificate_path" would be maybe long. renamed to "certificate"